### DISCUSSION TITLE
读取不到Cookies

### DISCUSSION THREAD

**Discussion by Lsw999160** [Yuukiy/JavSP#199] [问答 | Q&A] [answered]
报错：无法从浏览器Cookies文件获取JavDB的登录凭据([Errno 13] ，可能是安全软件在保护浏览器Cookies文件。
使用的是chrome正式版，edge也不行，要如何解决呢？

**Comment by Yuukiy** [comment, marked as answer]
这个功能只对JavDB的VIP有用。如果你是，传一下log
